test/test_transforms_v2_consistency.py

```diff
@@ -822,7 +822,7 @@ def test_random_choice(self, probabilities):
                 v2_transforms.Resize(256),
                 legacy_transforms.CenterCrop(224),
             ],
-            probabilities=probabilities,
+            p=probabilities,
```

Comment:
Nit: should we also rename the parameter to `p`?

```suggestion
            p=p,
```

Reply:
I think it is ok